.github/workflows/dapr-pubsub.yaml

```
@@ -0,0 +1,71 @@
+name: dapr-pubsub
+on:
+  push:
+    paths:
+      - "pkg/pubsub/dapr"
+      - "test/pubsub/**"
+  pull_request:
+    paths:
+      - "pkg/pubsub/dapr"
+      - "test/pubsub/**"
+permissions: read-all
+
+jobs:
+  dapr_test:
+    name: "Dapr pubsub test"
+    runs-on: ubuntu-22.04
+    timeout-minutes: 15
+    strategy:
+      matrix:
+        HELM_VERSION: ["3.7.2"]
+        GATEKEEPER_NAMESPACE: ["gatekeeper-system"]
+        DAPR_VERSION: ["1.10"]
+        AUDIT_CONNECTION: ["audit"]
+        AUDIT_CHANNEL: ["audit"]
+    steps:
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+
+    - name: Set up Go 1.19
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.19
+
+    - name: Bootstrap e2e
+      run: |
+        mkdir -p $GITHUB_WORKSPACE/bin
+        echo "$GITHUB_WORKSPACE/bin" >> $GITHUB_PATH
+        make e2e-bootstrap
+        make e2e-helm-install
+        helm repo add dapr https://dapr.github.io/helm-charts/
+        helm repo add bitnami https://charts.bitnami.com/bitnami
+        helm repo update
+        helm upgrade --install dapr dapr/dapr --version=${{ matrix.DAPR_VERSION }} --namespace dapr-system --create-namespace --wait --debug
+        helm install redis bitnami/redis --namespace default --wait --debug
```

do we want to pin redis to a version?

---

nit
```suggestion
        helm upgrade --install redis bitnami/redis --namespace default --wait --debug
```

so we align on using same convention